src/vs/workbench/contrib/terminal/browser/xterm/decorationAddon.ts

```diff
@@ -381,6 +381,16 @@ export class DecorationAddon extends Disposable implements ITerminalAddon {
 			if (actions.length > 0) {
 				actions.push(new Separator());
 			}
+			const labelCopyCommandAndOutput = localize("terminal.copyCommandAndOutput", 'Copy Command and Output');
+			actions.push({
+				class: undefined, tooltip: labelCopyCommandAndOutput, id: 'terminal.copyCommandAndOutput', label: labelCopyCommandAndOutput, enabled: true,
+				run: () => {
+					const text = command.getOutput();
+					if (typeof text === 'string') {
+						this._clipboardService.writeText(`${command.command !== '' ? command.command + '\n' : ''}${text}`);
```

Comment:
I'm actually not 100% sure if the check for `command.command !== ''` is necessary.

Reply:
I included it because this file does a similar check on [line 349](https://github.com/microsoft/vscode/pull/192217/files#diff-f26cf22b79d035fac45646c72f194cc92cf9629bd5cb58e8d8a39d220bb0d5dcR349).